docs/aspace_svn/readme.md

```diff
@@ -0,0 +1,13 @@
+This process sends ArchivesSpace records, serialized as EAD, to SVN once a month.
+
+### Summary
+The [get_eads](https://github.com/pulibrary/lib_jobs/blob/main/lib/tasks/lib_jobs.rake) task runs the [get_eads_job](https://github.com/pulibrary/lib_jobs/blob/main/app/models/aspace_svn/get_eads_job.rb
+). It is [scheduled](https://github.com/pulibrary/lib_jobs/blob/main/config/schedule.rb) to run on the first Saturday of the month.
+
+### Steps
+1. authenticate to SVN and ASpace
+2. create directory structure if it doesn't already exist
```

Comment:
Could you give more details on what the directory structure is supposed to look like? Is this done as a requirement for SVN or is it something relevant to how we organize the EADs?

Reply:
Done!